cg_hermes/config/balsamic.py

```diff
@@ -1,601 +1,738 @@
 """Tags that are defined in Balsamic deliverables mapped to tags used in CG
 
-The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not mandatory by default.
-However the tags that are available to a particular analysis is mandatory for that analysis.
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However the tags that are available to a particular analysis is mandatory for that analysis.
 """
 
+TAGS = {
+    # QC & reports
+    "multiqc_report.html": ["multiqc-html", "html"],
+    "multiqc_data.json": ["json", "multiqc-json"],
+    "report.html": ["balsamic-report"],
+    "config.json": ["balsamic-config"],
+    "BALSAMIC_X.X.X_graph.pdf": ["balsamic-dag"],
+    "metrics_deliverables.yaml": ["qc-metrics-yaml", "yaml"],
+    "concatenated_*_R_fastp.html": [
+        "quality-trimmed-fastq-html",
+        "html",
+    ],
+    "concatenated_*_R_fastp.json": [
+        "json",
+        "quality-trimmed-fastq-json",
+    ],
+    # Raw data & alignment files
+    "concatenated_*_R_1.fp.fastq.gz": [
+        "quality-trimmed-fastq-read1",
+        "read1",
+    ],
+    "concatenated_*_R_2.fp.fastq.gz": [
+        "quality-trimmed-fastq-read2",
+        "read2",
+    ],
+    "tumor.merged.cram": ["tumor-cram", "cram"],
+    "tumor.merged.cram.crai": ["tumor-cram-index", "cram"],
+    "normal.merged.cram": ["normal-cram", "cram"],
+    "normal.merged.cram.crai": ["normal-cram-index", "cram"],
+    # tnscope (WGS) & vardict (PANEL)
+    "tnscope.all.vcf.gz": ["tnscope", "vcf-all", "snv", "annotated-somatic-vcf-all"],
+    "tnscope.all.vcf.gz.tbi": ["tnscope", "annotated-somatic-vcf-all-index", "vcf-all", "snv"],
+    "tnscope.all.vcf.gz_summary.html": [
+        "tnscope",
+        "annotated-somatic-vcf-summary",
+        "vcf-summary",
+        "snv",
+    ],
+    "tnscope.balsamic_stat": ["stat-somatic-tmb", "snv", "tmb", "tnscope"],
+    "vardict.all.vcf.gz": ["annotated-somatic-vcf-all", "snv", "vardict", "vcf-all"],
+    "vardict.all.vcf.gz.tbi": ["snv", "vardict", "vcf-all", "annotated-somatic-vcf-all-index"],
+    "vardict.all.vcf.gz_summary.html": [
+        "snv",
+        "annotated-somatic-vcf-summary",
+        "vcf-summary",
+        "vardict",
+    ],
+    "vardict.balsamic_stat": ["tmb", "snv", "stat-somatic-tmb", "vardict"],
+    "tnscope_vardict.all.filtered.vcf.gz": ["vcf-filtered", "clinical-vcf-filtered", "snv"],
+    "tnscope_vardict.all.filtered.vcf.gz.tbi": [
+        "vcf-filtered",
+        "clinical-vcf-filtered-index",
+        "snv",
+    ],
+    "tnscope_vardict.all.filtered.pass.vcf.gz": ["clinical-vcf-pass", "vcf-pass", "snv"],
```

Comment:
```suggestion
    "tnscope_vardict.all.filtered.pass.vcf.gz": ["clinical-vcf-pass", "vcf-pass", "snv"],
```
I don't think we have any outputs with `tnscope_vardict`

Reply:
We don't, but BALSAMIC generates the same tags for both and since these caller don't overlap (vardict is executed for PANEL and tnscope for WGS) I have considered that we can leave it that way. If you think it's better that each one has its own tags, we can easily change it in BALSAMIC (that's why I also didn't close that [PR](https://github.com/Clinical-Genomics/BALSAMIC/pull/849)).

Reply:
I see. As this [PR ](https://github.com/Clinical-Genomics/BALSAMIC/pull/849) is also addressing adding specific tags to the varcaller. For readability, it will be better if you can assign these with their own tags.

Reply:
👍